components/raftstore/src/store/fsm/apply.rs

```diff
@@ -290,7 +290,7 @@ struct ApplyContext {
     apply_res: Vec<ApplyRes>,
     exec_ctx: Option<ExecContext>,
 
-    kv_wb: Option<WriteBatch>,
+    kv_wb: Option<RocksWriteBatch>,
```

Comment:
Could we replace `Arc<DB>` with trait `Engine`?

Reply:
I've started the work of converting all of raftstore to engine traits in https://github.com/tikv/tikv/pull/6707 and in another branch locally. This particular file has lots of dependencies so it might be one of the last to get converted.